apps/identity_web/lib/identity_web/router.ex

```diff
@@ -18,10 +18,18 @@ defmodule IdentityWeb.Router do
   scope "/", IdentityWeb do
     pipe_through [:browser]
 
-    get "/users/log_in", UserAuthController, :new
+    get "/users/auth", UserAuthController, :new
```

Comment:
Why did you change the route? Does this have any technical reason?

Reply:
This route is common to login and register.